webhook_features/src/features/summary_comment.rs

```diff
@@ -293,7 +293,7 @@ lazy_static! {
         (r"\b(NACK)\b", AckType::ConceptNack),
         (r"\b(Concept ACK)\b", AckType::ConceptAck),
         (
-            r"\b(?:re)?(ACK|utACK|tACK|crACK)(?:\s*)([0-9a-f]{6,40})\b",
+            r"\b(?:re|Re)?(ACK|utACK|tACK|crACK)(?:\s*)([0-9a-f]{6,40})\b",
```

Comment:
I wonder if this can be simplified to just (without the word boundary)

```
r"(ACK)(?:\s*)([0-9a-f]{6,40})\b"

Reply:
Yes, see https://github.com/MarcoFalke/DrahtBot/compare/c7ae8bfee2d5bca6875d16b9c794f78bf3a86e13..f3eb4fceb9a0866aac60596ce1adf0b3da13e017